### ISSUE TITLE
Update npm version to latest version

### ISSUE THREAD

**Issue by madhaze** [NUKnightLab/juxtapose#154]
`npm install --save juxtaposejs` looks to be pointing to `1.1.6` version.  
https://www.npmjs.com/package/juxtaposejs

Is there a specific reason for this?  Can this be updated to the latest version?

**Comment by blutorange** [comment]
Yeah, would be awesome if the latest version could be pushed to NPM! primefaces/primefaces#12717

**Comment by saitho** [comment]
Bump :)
